src/WebTestAssertionsTrait.php

```diff
@@ -217,10 +227,33 @@ public static function assertSelectorAttributeWillNotContain(string $locator, st
         self::assertSelectorAttributeNotContains($locator, $attribute, $text);
     }
 
+    private static function getText(string $locator): string
+    {
+        $client = self::getClient();
+        if ($client instanceof PantherClient) {
+            return self::findElement($locator)->getText();
+        }
```

Comment:
To preserve the current assertion behavior, I used `findElement` when using a `PantherClient` so this is still resolved via the `RemoteWebDriver` instead of using the Crawler. Same goes for `getAttribute`, `assertSelectorExists` and `assertSelectorNotExists`. By doing this, this should not be a breaking change.